### PR TITLE
Add react-native-basic-carousel

### PR DESCRIPTION
**Pull request by macvish**
- [x] Added `react-native-basic-carousel` library to **`react-native-libraries.json`**

### CODE REVIEW

File: react-native-libraries.json [hunk 1]
```diff
@@ -9069,5 +9069,16 @@
...
     "web": true,
     "expo": true
+  },
+  {
+    "githubUrl": "https://github.com/macvish/react-native-basic-carousel",
+    "npmPkg": "@mactony1/react-native-basic-carousel",
+    "images": ["https://github.com/macvish/react-native-basic-carousel/raw/develop/example/example.gif"],
+    "examples": [
+      "https://github.com/macvish/react-native-basic-carousel/tree/develop/example"
+    ]
```
Comment (Simek) [inline react-native-libraries.json:9079]:
```suggestion
    ],
```

Syntax issuer fix.

File: react-native-libraries.json [hunk 2]
```diff
@@ -9069,5 +9069,16 @@
     "android": true,
     "web": true,
     "expo": true
+  },
+  {
+    "githubUrl": "https://github.com/macvish/react-native-basic-carousel",
+    "npmPkg": "@mactony1/react-native-basic-carousel",
```
Comment (Simek) [inline react-native-libraries.json:9075]:
```suggestion
```

Isn't the package name just `react-native-basic-carousel` - https://www.npmjs.com/package/react-native-basic-carousel? If so this field can be skipped.

**Comment by macvish** [comment]
Thanks @Simek

**Review by Simek — APPROVED** [review]
Thank you for the quick updates! 👍

**Pull request merged**